### PR TITLE
Added link URL rewriting using a given host e.g. localhost or a local IP.

### PR DESCRIPTION
**Pull request by gon**

### CODE REVIEW

File: neo-proxy.js [hunk 1]
```diff
@@ -11,6 +11,17 @@ function neoProxy(options) {
...
+
+  //
+  // Rewrite URLs of HTML link elements with your local host IP
+  //
+  var rewriteUrlsScript = ''
+  if (options.rewriteUrlsHost) {
+    rewriteUrlsScript = "<script>"
+    rewriteUrlsScript += rewriteUrlsFunction + "; rewriteUrlsFunction('" + options.target + "', '" + options.rewriteUrlsHost + "', " + options.port + ");"
+    rewriteUrlsScript += "</script>"
+  }
```
Comment (DTrejo) [inline neo-proxy.js:24]: maybe we should pull in http://ghub.io/multiline ? thoughts?

File: neo-proxy.js [hunk 2]
```diff
@@ -95,3 +106,12 @@ function neoProxy(options) {
 
   return options
 }
+
+function rewriteUrlsFunction (host, localHost, port) {
+  var links = document.getElementsByTagName('a');
+  for (var i = links.length - 1; i >= 0; i--) {
+    var link = links[i],
+      href = link.getAttribute('href');
```
Comment (DTrejo) [inline neo-proxy.js]: one var per line is a good idea (explanation: http://benalman.com/news/2012/05/multiple-var-statements-javascript/ )

File: neo-proxy.js [hunk 3]
```diff
@@ -95,3 +106,12 @@ function neoProxy(options) {
...
+    if (href && href.indexOf(host) > -1 ) { link.setAttribute('href', href.replace(host, 'http://' + localHost + ':' + port) + '?host=' + host) };
+  };
```
Comment (DTrejo) [inline neo-proxy.js:118]: this is worth making multiple lines

File: readme.md [hunk 4]
```diff
@@ -28,7 +28,8 @@ npm install neo-proxy --save
     port: 8888,
     target: 'http://dtrejo.com',
     needle: '</body>',
-    customHTML: '<script>alert("Hello, it's working! — DTrejo")</script>'
+    customHTML: '<script>alert("Hello, it's working! — DTrejo")</script>',
+    rewriteUrlsHost: 'localhost' // or '10.1.1.250'
```
Comment (DTrejo) [inline readme.md:32]: - Could you write a sentence or two about the purpose of rewriteUrlsHost? Why someone would want to use it, how to disable it, etc.
- Maybe change the name of the option so it is more obvious?

While you're in here, would you remove the `it's` contraction? The quote makes it invalid JS, my fault.

File: neo-proxy.js (hunk 1 above)
Comment (gon) [inline neo-proxy.js:24]: Let's hold off until ES6 and `template strings`.

**Pull request merged**